### ISSUE TITLE
Cannot specify 0 as localization coordinate (x,y,z) 

### ISSUE THREAD

**Issue by kidanger** [cmla/rpcm#10]
I am trying to use the following command:
```bash
$ rpcm localization image.tif -x 200 -y 200 -z 0
usage: rpcm [-h] {projection, localization, crop, footprint} ...
rpcm: error: either --points or {-x, -y, -z} must be defined
```
`-z` is considered not set since it equals to 0, yet 0 should be a valid value.

https://github.com/cmla/rpcm/blob/ad662561a05aef9dea1b47e28dc451f5ee48c4ae/rpcm/cli.py#L118
https://github.com/cmla/rpcm/blob/ad662561a05aef9dea1b47e28dc451f5ee48c4ae/rpcm/cli.py#L120

Bonus question: is there a way to not specify `z` and let rpcm use SRTM instead?

**Comment by carlodef** [comment]
Thank you @kidanger, this was indeed a bug. It should be fixed in commit e2abb4c.

About the bonus question, one can for sure use a DEM such as SRTM to get a `z` value when calling the _projection_ function `lon, lat, z` --> `col, row`. In this case `z` can simply be read from the DEM at position `lon, lat`. 

But for the _localization_ function `col, row, z` --> `lon, lat` it's a bit more complex as one doesn't know beforehand where to evaluate the DEM to get `z`. A possible solution is to solve the equation `DEM(localization(col, row, z)) = z` for `z`, which amounts to finding a fixed point for the function `z` --> `DEM(localization(col, row, z))`. We could try with https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.fixed_point.html.

**Comment by kidanger** [comment]
Thanks, it works now.

Right, I forgot that it was not trivial for localization, so I guess that's out of scope.
